awscli/examples/rum/list-tags-for-resource.rst

```diff
@@ -0,0 +1,19 @@
+**To display the tags associated with a CloudWatch RUM resource**
+
+The following ``list-tags-for-resource`` displays the tags associated with a CloudWatch RUM resource. ::
+
+	aws rum list-tags-for-resource \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general